tests/TracingTest.php

```diff
@@ -15,26 +15,26 @@ class TracingTest extends TestCase
 {
     public function testContextGenerationAndRestore()
     {
-        $spanContext = SpanContext::generate();
```

Comment:
changed these to snake_case to appease PHPCS; another thing I think we need to discuss